prime-router/settings/STLTs/DE/de-dph.yml

```diff
@@ -0,0 +1,114 @@
+- name: "de-dph"
+  description: "Delaware Division of Public Health"
+  jurisdiction: "STATE"
+  stateCode: "DE"
+  countyName: null
+  filters:
+  - topic: "full-elr"
+    jurisdictionalFilter:
+    - "orEquals(ordering_facility_state, DE, patient_state, DE)"
```

Comment:
This is covid filter.  Should it convert to fhirpath?

Reply:
These filters look like they are the organizational filters. I don't know if removing this section will remove the organizational filters for the covid-19 receiver. I would just leave the original topic `covid-19` since those filters are still applicable to the covid-19 receiver.